### ISSUE TITLE
contao:backup:create should not delete different backups

### ISSUE THREAD

**Issue by Metis77** [contao/contao#6185]
### Description

With `console contao:backup:create` I backup several DB states an keep them following the `keep_intervals` setting from the `config.yml`.

At the same time I push an pull the DB while deployment from local to server, but use different backup filenames for this.
For example I run `console contao:backup:create db_from_server__00000000000000.sql.gz` .

Problem:
Each `contao:backup:create` call deletes all backups, especially the ones with different filenames than the one given in CLI.

Solution / Feature Request:
`contao:backup:create` should ignore all backup files not matching the given (if empty the default) backup file naming.

**Comment by ausi** [comment]
Shouldn’t this be seen as a bug rather than a feature?

**Comment by Metis77** [comment]
> Shouldn’t this be seen as a bug rather than a feature?

yes, probably this should be considered as a bug.

**Comment by Toflar** [comment]
I don't consider this a bug. You cannot create a backup with an invalid filename. It's validated. Actually if `00000000000000` works, this should be considered a bug. But backups are entirely managed by Contao and it's absolutely correct that yours gets deleted.

If we wanted to support your use case of backups that are not deleted, then what should be done instead is allowing `contao:backup:create` to accept a path **outside** of the managed directory. But as long as you only provide a different name, I don't think it should be ignored.

**Comment by ausi** [comment]
So something like this?

```sh
console contao:backup:create --path=var/my-custom-backups/db_from_server.sql.gz
```

**Comment by Metis77** [comment]
> So something like this?
> 
> ```shell
> console contao:backup:create --path=var/my-custom-backups/db_from_server.sql.gz
> ```

that should be fine.
I think the important thing for pushing/pulling db from/to Server is a concrete filename without dynamic date.

**Comment by fritzmg** [comment]
Filename based retention policies are pretty common in back up softwares though.

**Comment by Toflar** [comment]
Yeah whatever. I think the current behavior is absolutely correct and we don't need to do anything. If anybody wants to add support to be able to create a backup **outside** of the managed backup directory, then I'm fine with that. But that's definitely a feature and not a bug.
